storage/engine/mvcc.go

```diff
@@ -1097,15 +1164,17 @@ func MVCCFindSplitKey(engine Engine, raftID int64, key, endKey proto.Key) (proto
 // used after a range is split to recompute stats for each
 // subrange. The start key is always adjusted to avoid counting local
 // keys in the event stats are being recomputed for the first range
-// (i.e. the one with start key == KeyMin).
-func MVCCComputeStats(engine Engine, key, endKey proto.Key) (MVCCStats, error) {
+// (i.e. the one with start key == KeyMin). The elapsedNanos arg
+// specifies the wall time in nanoseconds since the epoch and is used
```

Comment:
The name "elapsedNanos" sounds like a duration, not a time since the epoch. Is this the current time when MVCCComputeStats is called, or is it the time of some other event?